### PR TITLE
Simulation split

### PR DESCRIPTION
**Pull request by karabowi**
MQ/examples/simulation_split

First version of the MC simulation in MQ, where
generated events are sent to transport devices.

---

Checklist:

* [ ] Rebased against `dev` branch
* [ ] My name is in the resp. CONTRIBUTORS/AUTHORS file
* [ ] Followed [the seven rules of great commit messages](https://chris.beams.io/posts/git-commit/#seven-rules)

### CODE REVIEW

File: examples/MQ/simulation_split/run/runMQTrans.cxx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    detArray->Add(cave);
+    // Pixel*  det = new Pixel("PixelDetector", kTRUE);
+    // det->SetGeometryFileName("pixel.geo");
+    // detArray->Add(det);
+    run->SetDetectorArray   (detArray);
+
+    run->SetStoreTraj       (false);
+
+    // ---------------------------------------------------
+    if ( 1==0 )
```
Comment (rbx) [inline examples/MQ/simulation_split/run/runMQTrans.cxx]: why do you need this?

**Comment by karabowi** [comment]
I guess you mean the if (1==0).

tldr: it should show the end user possible features.

in principle you are right. 
I got the piece of code when I copied from pixelDetector/run/runMQSim.cxx
where you would probably ask exactly the same question (I am the author of that code either).

The lines included in the if will be immediately clear to anyone who is running any simulations in FairRoot/PandaRoot/Cbmroot.
It simply shows the user, that s/he can attach to FairMQSimDevice or FairMQTransportDevice
any FairTask that s/he may deem necessary to run together with the particle transport.
It is sometimes necessary, but in FairRoot's simple detector examples is not needed.

yours,
radek

**Comment by rbx** [comment]
> it should show the end user possible features.

Perhaps you can hide it behind a cmd argument that is off by default, then it is easily usable without recompiling

**Comment by dennisklein** [comment]
Hi, a rebase against current `dev` should restore alfa-ci checks (The `dev_mt` check on Linux is failing at the moment).
